private/buf/bufcli/cache.go

```diff
@@ -96,6 +98,11 @@ var (
 	//
 	// Normalized.
 	v3CacheWKTRelDirPath = normalpath.Join("v3", "wellknowntypes")
+	// v3CacheModuleLockRelDirPath is the relative path to the lock files directory for module data.
+	// This directory is used to store lock files for synchronizing reading and writing module data from the cache.
+	//
+	// Normalized.
+	v3CacheModuleLockRelDirPath = normalpath.Join("v3", "module_locks")
```

Comment:
`wellknowntypes` doesn't use underscores, this shouldn't either, for consistency - `modulelocks`